start.py

```diff
@@ -224,6 +224,14 @@ def activate_license():
             prefs_file.write(prefs_body)
 
 
+def get_application_root_url(vcap_data):
+    try:
+        return "https://%s" % vcap_data["application_uris"][0]
```

Comment:
We are trying to use the `.format` syntax instead of string interpolation going forward, like this:

```suggestion
        return "https://{}".format(vcap_data["application_uris"][0])
```